app/views/defence_requests/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= flash_messages %>
 
-<h1><%= "#{t(current_user.role)}  #{t('dashboard')}" %></h1>
+<h1><%= "#{t(current_user.roles.first.to_s)}  #{t('dashboard')}" %></h1>
```

Comment:
Absolutely. I was planning on leaving that off until getting started on actually consuming profile info from the Auth app. This PR is a stop gap/setup of sorts, I guess.